apps/src/templates/VersionHistory.jsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/order */
+// Disabling import order due to circular dependencies.
```

Comment:
Could you expand on this comment a bit more? I'd like to capture information here about what the specific circular dependency is and exactly what about the code would need to change for us to no longer need this.

If possible, I'd also ideally like to only ignore this rule for the specific imports that need to ignore it, and to still apply it to all other imports in the file.